private-templates-service/client/src/components/Designer/Designer.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useEffect } from 'react';
+import { RootState } from '../../store/rootReducer';
+import requireAuthentication from '../../utils/requireAuthentication';
+import { connect } from 'react-redux';
+import { UserType } from '../../store/auth/types';
+import { useHistory } from 'react-router-dom';
+//ACDesigner
+import * as monaco from 'monaco-editor';
+import markdownit from 'markdown-it';
+import * as ACDesigner from 'adaptivecards-designer';
+
+const mapStateToProps = (state: RootState) => {
+  return {
+    isAuthenticated: state.auth.isAuthenticated,
+    user: state.auth.user
+  };
+};
+
+interface DesignerProps {
+  isAuthenticated: boolean;
+  user?: UserType;
+  authButtonMethod: () => Promise<void>;
```

Comment:
I still dont know why this is passed everywhere?

Reply:
True, I'll take out authButtonMethod but I think we still need the other two props